public/pages/AnomalyCharts/containers/AnomaliesChart.tsx

```diff
@@ -256,183 +177,129 @@ export const AnomaliesChart = React.memo((props: AnomaliesChartProps) => {
-                minValue={anomalySummary.minConfidence}
-                maxValue={anomalySummary.maxConfidence}
-                description={
-                  props.showAlerts ? 'Confidence' : 'Sample confidence'
-                }
-                tooltip="Indicates the level of confidence in the anomaly result."
-              />
-            </EuiFlexItem>
-            <EuiFlexItem>
-              <EuiStat
-                title={
-                  props.isLoading || isLoadingAlerts
-                    ? ''
-                    : anomalySummary.lastAnomalyOccurrence
-                }
-                description={
-                  props.showAlerts
-                    ? 'Last anomaly occurrence'
-                    : 'Last sample anomaly occurrence'
-                }
-                titleSize="s"
-              />
-            </EuiFlexItem>
-            {props.showAlerts ? (
-              <EuiFlexItem>
-                <AlertsStat
-                  monitor={props.monitor}
-                  showAlertsFlyout={() => setShowAlertsFlyout(true)}
-                  totalAlerts={totalAlerts}
-                  isLoading={props.isLoading}
-                />
+          {props.isHCDetector &&
+          props.onHeatmapCellSelected &&
+          props.detectorCategoryField ? (
+            <EuiFlexGroup style={{ padding: '20px' }}>
+              <EuiFlexItem style={{ margin: '0px' }}>
+                <div
+                  style={{
+                    width: '100%',
+                    opacity: showLoader ? 0.2 : 1,
+                  }}
+                >
+                  {props.isLoading ? (
+                    <EuiFlexGroup
+                      justifyContent="spaceAround"
+                      style={{ paddingTop: '150px' }}
+                    >
+                      <EuiFlexItem grow={false}>
+                        <EuiLoadingChart size="xl" mono />
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  ) : (
+                    [
+                      <AnomalyHeatmapChart
+                        detectorId={props.detectorId}
+                        detectorName={props.detectorName}
+                        dateRange={props.dateRange}
+                        //@ts-ignore
+                        title={props.detectorCategoryField[0]}
+                        anomalies={anomalies}
+                        isLoading={props.isLoading}
+                        showAlerts={props.showAlerts}
+                        monitor={props.monitor}
+                        detectorInterval={props.detectorInterval}
+                        unit={props.unit}
+                        onHeatmapCellSelected={props.onHeatmapCellSelected}
+                      />,
+                      props.showAlerts === undefined || !props.showAlerts
+                        ? [
+                            <EuiSpacer size="m" />,
+                            <AnomalyOccurrenceChart
```

Comment:
`AnomalyOccurrenceChart` is only used by HC detector and `AnomalyDetailChart` is only used single entity detector?

Reply:
Yes, and `AnomalyOccurrenceChart ` contains `AnomalyDetailChart`. `AnomalyOccurrenceChart` is actually wrapper of `AnomalyDetailChart`, making `AnomalyDetailChart` a separate chart so that it can be easily re-used in multiple places